### PR TITLE
🇪🇹 Update on candidate lock ETA

### PR DESCRIPTION
**Pull request by WRadoslaw**
- #3619

### CODE REVIEW

File: packages/ui/src/accounts/components/AccountItem/components/lockItems/CouncilCandidateLockItem.tsx [hunk 1]
```diff
@@ -25,26 +27,41 @@ export const CouncilCandidateLockItem = ({ lock, address, isRecoverable }: LockD
...
   const electionId = eventData?.candidate.electionRoundId
-  const electionStart = eventData?.candidate.electionRound.referendumStageVoting?.createdAt
   const voteStageDuration = api?.consts.referendum?.voteStageDuration.toNumber()
   const revealStageDuration = api?.consts.referendum?.revealStageDuration.toNumber()
 
   const recoveryTime = useMemo(() => {
-    if (!electionStart || !voteStageDuration || !revealStageDuration) {
-      return
+    if (!voteStageDuration || !revealStageDuration || !remainingPeriod) {
+      return { unrecoverableLabel: 'Recoverable after failed election' }
```
Comment (thesan) [inline packages/ui/src/accounts/components/AccountItem/components/lockItems/CouncilCandidateLockItem.tsx:42]: Shouldn't it be something like: "Recoverable after lost election or end of the council term" ?

**Review by thesan — CHANGES_REQUESTED** [review]

File: packages/ui/src/accounts/components/AccountItem/components/lockItems/CouncilCandidateLockItem.tsx (hunk 1 above)
Comment (WRadoslaw) [reply packages/ui/src/accounts/components/AccountItem/components/lockItems/CouncilCandidateLockItem.tsx:42]: Not really, council candidate lock can only be withdrawn after the lost election, if one wins the election then the lock transforms to a councilor lock which is handled by another list item.

This other list item for Councilor has the following tooltip:
image

**Comment by dmtrjsg** [comment]
"Recoverable after lost election or end of the council term" 👍

**Review by thesan — APPROVED** [review]
Please retarget to `dev` once #3640 gets merged

**Pull request merged**